lit_tests/kernel/wave/expansion.py

```diff
@@ -106,21 +106,21 @@ def test_read_write_equal_sizes():
         # CHECK-NEXT: placeholder(_name=a
         # CHECK-NEXT: placeholder(_name=c
         # CHECK-NEXT: read(memory=a
-        # CHECK-SAME: index={M: BLOCK_M*WG0, N: BLOCK_N*WG1}
+        # CHECK-SAME: index={M: $WG0*BLOCK_M, N: $WG1*BLOCK_N}
```

Comment:
Interesting, I assume sympy sorts vars in commutative exprs based on name.

Reply:
yeah I was surprised by that as well.